scripts/create-kubestellar-demo-env.sh

```diff
@@ -28,7 +56,12 @@ kubestellar_version=0.25.1
 echo -e "KubeStellar Version: ${kubestellar_version}"
 
 echo -e "Checking that pre-req softwares are installed..."
-curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
+if [ "$k8s_platform" == "kind" ]; then
+    curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
+else
+    ../hack/check_pre_req.sh -V kflex ocm helm kubectl docker k3d
```

Comment:
(A) using a local file does not work when this script is used in curl-to-bash style.

(B) This particular way of navigating to a local file adds an undocumented assumption about current working directory.

Reply:
You could make this testable before merging by curl-to-bash of https://raw.githubusercontent.com/clubanderson/kubestellar/refs/heads/add-k3d-to-create-demo-env/hack/check_pre_req.sh

Reply:
that worked - will check in the update